### DISCUSSION TITLE
Starter inconsistent behavior with /BCS/WALL

### DISCUSSION THREAD

**Discussion by marcoserra68** [OpenRadioss/OpenRadioss#2734] [Users]
Hello all,

A new /BCS/WALL capability was recently introduced that is compatible with LAW151. It allows the creation of wall boundary conditions within the domain of an Euler mesh by selecting the nodes on the vertices of an element face (or edge in 2D). I have been running a small test problem, supplied by the Radioss developers, that uses this capability and it runs without any issues. I have attached a manual page describing this card.

I have generated my own test model that implements the same /BCS/WALL cards in the same way, but this model will not make it through the Starter, ending with a somewhat generic Error 760.  Without the /BCS/WALL cards, the model goes through the Starter and runs, so it seems as if the /BCS/WALL cards are the problem.

Any ideas/suggestions as to why the use of the same cards in two different models might generate this error 760?

Thank you in advance.
Marco Serra

[BCS_WALL.pdf](https://github.com/user-attachments/files/15775650/BCS_WALL.pdf)

**Comment by Schwoertzig** [comment]
Hi @marcoserra68 ,
is it possible to share the input file ? Or even a small reproducer ?
Thierry

**Reply by marcoserra68** [reply]
Hello Thierry (@Schwoertzig) 

I have attached the two models I have described:

Model 1 (TPZY_0000.rad, TPZY_0001.rad) is the model that was generated as a test case for the /BCS by Radioss developers. This one I can run without any issue.

Model 2 (cptl-02_0000.rad, ADDCARDS.inc, cptl-02_0001.rad) is my test model that I am having trouble with.

[models.tar.gz](https://github.com/user-attachments/files/15775944/models.tar.gz)

Thank you.
Marco

**Reply by Schwoertzig** [reply]
There is a missing character on detonation point :
![image](https://github.com/OpenRadioss/OpenRadioss/assets/104987175/04535dbc-48ea-459f-9c46-876e6334e54b)
After that, input file runs correctly from my side :
![image](https://github.com/OpenRadioss/OpenRadioss/assets/104987175/d7f503d4-1c06-47bc-a951-fa0eec4a026a)
![image](https://github.com/OpenRadioss/OpenRadioss/assets/104987175/b2eab2a5-802f-4e0d-8f50-301be8abc1fe)

**Reply by Schwoertzig** [reply]
What is strange is that I did not reproduced your **error id 760**.

With missing character in detonation point I had **error id 100103** : 
![image](https://github.com/OpenRadioss/OpenRadioss/assets/104987175/81da6389-e300-4c61-8d76-11dc88a28534)

Could you please share your Starter listing file : cptl-02_0000.out ?

**Reply by marcoserra68** [reply]
Thank you Thierry,

I'm sorry, I must have deleted the character by mistake. Before I shared the file I had to remove some information that I could not share...

Anyway, after putting the # character back in, I ran it again exactly as I sent it to you and I have attached the .out file.

[cptl-02_0000.out.gz](https://github.com/user-attachments/files/15776981/cptl-02_0000.out.gz)

Hopefully this helps.

Just so you know, I am quite new to OpenRadioss, so I am not yet intimately familiar with all of the ins and outs of it. I have used LS-Dyna for many years.

Thank you.
Marco

**Reply by Schwoertzig** [reply]
Hi @marcoserra68,

I cannot reproduce the issue; it might be related to the memory state on your computer. We should consider using a memory debugger or testing on other platforms.

FYI, I will be unavailable for one week. If no one else responds, I will take a look when I return.

**Reply by marcoserra68** [reply]
Hi Thierry,

Thank you so much for your efforts to help me so far. FYI, I have tried to run this on another machine, but I get the same result. I will continue to look into it and try to understand in the meantime.

Thank you
Marco

**Reply by marcoserra68** [reply]
Hello Thierry,

I realize you are away, so I am putting this note here while it is still fresh in my mind.

I just installed OpenRadioss on a Windows 11 virtual machine running on my laptop and ran the model. It ran. It seems to me the issue most likely lies with my Linux machine(s) setup. I have tried to run it on Rocky 9.2 as well as Centos 7.2 (at some point to be upgraded to Rocky 9.2 as well). Both times failed with Error 760. Incidentally, I have run other models (much larger) on both machines without problems (OpenRadioss 2023).

Regards
Marco

**Reply by marcoserra68** [reply]
Hello Thierry,

I began to compare the problem model file with the test model file and started to add the commands that my problem model did not have that the test model did. I found that if I add the /IOFLAG command and set it to 6, as in the test model, then the Error 760 goes away. I don't have an explanation for this though.

Best regards
Marco

**Reply by Schwoertzig** [reply]
Hi @marcoserra68 ,
what is the status about this topic, is it still an open issue with the last revision of OpenRadioss ?
Regards

**Reply by marcoserra68** [reply]
Hi Thierry,

I just ran the starter on my test problems without the /IOFLAG workaround and it seems to work just fine. I will mark this as resolved.
Thank you.

Regards
Marco
